flutter/lib/src/default_integrations.dart

```diff
@@ -5,6 +5,10 @@ import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 import 'package:sentry/sentry.dart';
+// TODO is there a "semi-internal" export we can use instead?
+//      sentry_io.dart looks like that but doesn't have any comments.
+// ignore: implementation_imports
+import 'package:sentry/src/platform/platform.dart';
```

Comment:
You can probably look into:

`options.platformChecker.platform.isAndroid`, etc...
Since you have the instance of `SentryFlutterOptions`, you don't need to take the `platform` as a param.

Reply:
removed